gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-file-upload/api-documentation-v4-file-upload.component.ts

```diff
@@ -18,8 +18,9 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 import { NewFile } from '@gravitee/ui-particles-angular';
 
 import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+import { PageType } from '../../../../../entities/management-api-v2/documentation/pageType';
 
-const allowedFileExtensions = ['md', 'markdown', 'txt'] as const;
+const allowedFileExtensions = ['md', 'markdown', 'txt', 'json', 'yml', 'yaml'] as const;
```

Comment:
Looks like it isn't used anymore?

Reply:
Yes it was, but not properly. I fixed it.